src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  list-style-type: none;
+  margin-right: 50px;
+}
+
+.nav__link {
+  margin-right: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
```

Comment:
pay attention on reference and compare with test
link should be on full height of header
add vertical paddings for fix it
![Screenshot from 2023-04-11 14-40-01](https://user-images.githubusercontent.com/33029864/231152151-be04ed03-e30c-462e-be99-58cbc1eda2db.png)

Reply:
Should I add vertical paddings for class="computer__link" or for header ? there is mockup 22 and 23 so horizontal should be 0 0 ?